script.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable linebreak-style */
+const menuBox= document.getElementById('menu');
+const menuBar= document.getElementById('menu_bar');
+const closingBtn = document.getElementById('btn');
+const menuItems = document.getElementsByClassName('h-menu');
+const menuContainer = document.getElementsByClassName('menu-items');
+
+menuBar.onclick= function() {
```

Comment:
- [ ] I would advise you to add event by using the addEventListener which javascript provides. This is part of the best practices and you can find more on that [here](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)
- [ ] I would also recommend you to use arrow function instead of the function keyword. The arrow function is ES6 which shows you are updated on the latest version usage of JS. You can read on it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)